server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/GetMailboxesMethod.java

```diff
@@ -121,31 +125,46 @@ private GetMailboxesResponse getMailboxesResponse(GetMailboxesRequest mailboxesR
             .orElseGet(Throwing.supplier(() -> retrieveAllMailboxes(mailboxSession)).sneakyThrow());
     }
 
+    private MailboxFactory.MailboxBuilder getBuilder(
```

Comment:
please don't, you defeat the goal of the builder: having readable argument passing at calling site